persistence/src/test/java/edu/unc/lib/dl/persist/services/edit/EditTitleServiceTest.java

```diff
@@ -106,8 +115,12 @@ public void editTitleTest() throws Exception {
 
         verify(operationsMessageSender).sendUpdateDescriptionOperation(anyString(), pidsCaptor.capture());
         Collection<PID> pids = pidsCaptor.getValue();
-        assertEquals(pids.size(), 1);
+        assertEquals(1, pids.size());
         assertTrue(pids.contains(pid));
+
+        verify(contentObj).setDescription(inputStreamCaptor.capture());
+        Document updatedDoc = getUpdatedDescriptionDocument(inputStreamCaptor.getValue());
+        assertTrue(assertTitleValue(updatedDoc, title));
```

Comment:
I'd suggest renaming `assertTitleValue` to `hasTitleValue`. Alternatively, you could do the asserting inside of `assertTitleValue`, but you would need `assertTitleNotPresent` or similar for the negative case.